en/website.json

```diff
@@ -660,5 +660,7 @@
     "billing_period": "Billing Period",
     "uninvited_subscription": "Uninvited Subscription",
     "unknown": "Unknown",
-    "upgrade": "Upgrade"
+    "upgrade": "Upgrade",
+    "protection_role_search": "SEARCH BY ROLE ID OR NAME",
+    "protection_type_here": "Type here .."
```

Comment:
no need to specify "protection" in the key if the string can be used somewhere else

Reply:
fixed (2d27b80a5a2471650826c347411b25ab2edf3f71)